inst/include/population_dynamics/population/population.hpp

```diff
@@ -120,7 +119,8 @@ namespace fims_popdy {
 
             // size all the vectors to length of nages
             nfleets = fleets.size();
-            expected_catch.resize(nyears * nfleets);
+            total_landings_weight.resize(nyears);
```

Comment:
is there another type of landings besides "total"? or there also a fleet_landings_weight, etc.?

Reply:
There are landings by weight by fleet in the fleet module. This is total landings summed across all fleets. We could actually remove this, I'm not sure why landings are double tracked in the population and fleet modules. Seems like it just adds opportunity for errors.